gapii/cc/vulkan_extras.cpp

```diff
@@ -467,6 +467,8 @@ void VulkanSpy::recordFenceReset(CallObserver*, uint64_t) {}
 void VulkanSpy::recordAcquireNextImage(CallObserver*, uint64_t, uint32_t) {}
 void VulkanSpy::recordPresentSwapchainImage(CallObserver*, uint64_t, uint32_t) {
 }
+void VulkanSpy::recordBeginCommandBuffer(CallObserver*, uint64_t) {}
```

Comment:
We should start considering if there is a better way to handle forward dependencies without externs. 
In general, we should be trying to remove extens, but I understand in this case. It would be nice to not have to keep on adding them though

Reply:
Is there a way to represent `interface{}` in the api language? I think that would allow all of the forward dependency externs to be reduced to a set of simple wrappers around the three `watcher.*ForwardDependency()` functions. 

Alternatively, we could add language primitives to wrap these three functions, but it doesn't really feel like these belong at the language level.